src/elementPlugins/dropdownChoice/src/ApplicationView.tsx

```diff
@@ -0,0 +1,63 @@
+import React, { useState, useEffect } from 'react'
+import { Dropdown, Header, Message } from 'semantic-ui-react'
+import { ApplicationViewProps } from '../../types'
+
+const ApplicationView: React.FC<ApplicationViewProps> = ({
+  templateElement,
+  onUpdate,
+  initialValue,
+  isEditable,
+  currentResponse,
+  validationState,
+  onSave,
+}) => {
+  const [value, setValue] = useState(initialValue?.text)
+  const {
+    label,
+    description,
+    placeholder,
+    options,
+    default: defaultIndex,
+  } = templateElement?.parameters
+
+  useEffect(() => {
+    onUpdate(value)
+  }, [])
+
+  function handleChange(e: any, data: any) {
+    const { value } = data
+    console.log('Event', data)
+    onUpdate(value)
+    setValue(value)
+    onSave({ text: value, optionIndex: options.indexOf(value) })
+  }
+
+  const dropdownOptions = options.map((option: any, index: number) => {
+    return {
+      key: option,
+      text: option,
+      value: option,
+      index,
+    }
+  })
+
+  return (
+    <>
+      <Header content={label} />
+      {description}
+      <Dropdown
+        fluid
+        selection
+        defaultValue={options[defaultIndex]}
```

Comment:
Don't know what could be wrong but this is not working as expected.
Before you open the dropdown it doesn't show the default value:
<img width="695" alt="Screen Shot 2020-11-24 at 2 24 27 PM" src="https://user-images.githubusercontent.com/16461988/100038833-29a42880-2e69-11eb-8994-6b8ddef56ec8.png">

Then after it opens the dropdown it is showing the default and then all the options after:
<img width="519" alt="Screen Shot 2020-11-24 at 2 24 38 PM" src="https://user-images.githubusercontent.com/16461988/100038865-2f9a0980-2e69-11eb-950c-a3c8c11631c7.png">

Reply:
Yes, good point. I've made an change now to set the `value` to the default on load. I'm not actually sure what the `defaultValue` prop does, it seems like whatever the "value" prop is determines what shows. 

Also, the back-end didn't actually have a default set for this question, and I've made it so there's nothing pre-selected if no default (as opposed to the first item). I think this is the correct behaviour, since we need to be able to have nothing selected sometimes to force the user to make a conscious choice.